OpenRA.Game/Graphics/TargetLineRenderable.cs

```diff
@@ -35,7 +35,13 @@ public TargetLineRenderable(IEnumerable<WPos> waypoints, Color color, int width
 		public bool IsDecoration => true;
 
 		public IRenderable WithZOffset(int newOffset) { return new TargetLineRenderable(waypoints, color); }
-		public IRenderable OffsetBy(WVec vec) { return new TargetLineRenderable(waypoints.Select(w => w + vec), color); }
+
+		public IRenderable OffsetBy(in WVec vec)
+		{
+			var vec1 = vec;
```

Comment:
Doesn't this defy the pass by reference? But assume needed for the LINQ statement.

Reply:
Corroect, you cant use the `in` modifier in lambdas.